helper/testhelpers/testhelpers.go

```diff
@@ -457,3 +458,47 @@ func RaftClusterJoinNodes(t testing.T, cluster *vault.TestCluster) {
 
 	WaitForNCoresUnsealed(t, cluster, 3)
 }
+
+func GenerateDebugLogs(t testing.T, stopCh chan struct{}, client *api.Client) {
+	t.Helper()
+
+	// This ticker value was chosen after some trial and error trying to figure
+	// out why the ticker wasn't stopping when stopCh was closed. It turns out
+	// there's some kind of delay between closing a channel in a different
+	// goroutine and detecting that it was closed in this one. This ticker interval
+	// turned out to be the right value in combination with a 4 second delay
+	// in my other goroutine to let this helper produce log output. I'm not sure
+	// if I'm doing something wrong here, or if that's just the way this goes.
+	ticker := time.NewTicker(1500 * time.Millisecond)
+	var err error
+	var m sync.Mutex
```

Comment:
What does this mutex protect?

Reply:
There were data race failures in CI coming from this helper method.  The best I could tell, `client.Sys().Unmount()` didn't like being called from multiple goroutines, so this was my attempt at stopping those failures.